storage/mongo.go

```diff
@@ -157,23 +160,36 @@ func (r *Repository) AddDocument(userID string, ID string) error {
 func (r *Repository) AddVacation(ID string, userID string, startDate int64, durationHours float32) (entities.Vacation, error) {
 	session := r.session.Copy()
 	defer session.Close()
```

Comment:
does `session.Close()` return an error? If so we probably ought not to ignore it? Here is some nice pattern for this https://www.joeshaw.org/dont-defer-close-on-writable-files/

Reply:
`mgo.session.Close()` does not return error, but link is helpful to understand that such errors should be handled in other cases :thumbsup:

Reply:
As example, refactored grpc.ClientConn:
``` defer conn.Close()```
to
```
defer func() {
		if err := conn.Close(); err != nil {
			log.Fatalf("failed to close connection: %s", err)
		}
	}()
```